src/shared/card/ArticleFeedbackCard.js

```diff
@@ -98,7 +98,7 @@ const ArticleFeedbackCard = props => {
 
     setLikeEnabled(!likeEnabled);
     setDislikeDisable(!dislikeDisable);
-
+    setSubmitBtn(!submitBtn);
     setDislikeEnabled(false);
   };
   //   const hideFunction = () => {
```

Comment:
either remove unused codes or add a TODO when and how will it be removed or reused